collections/validator/check-if-an-array-contains-a-value-matching-some-criterias.md

```diff
@@ -1,8 +1,14 @@
 ~~~ javascript
 const contains = (arr, criteria) => arr.some(v => criteria(v));
 
+//or
+const contains = (arr, criteria) => arr.some(criteria);
+
+//or
+const contains = (arr,criteria) => arr.filter(criteria).length > 0
```

Comment:
Great. Can you change `//or` to `// Or` to make it consistent with other snippets?

Reply:
I changed the comment as you said